sdks/java/io/kafka/pom.xml

```diff
@@ -30,7 +30,7 @@
   <description>Library to read Kafka topics.</description>
 
   <properties>
-    <kafka.clients.version>0.9.0.1</kafka.clients.version>
+    <kafka.clients.version>0.10.1.0</kafka.clients.version>
```

Comment:
Yes, supposed to be. The binary built with this dependency should work fine even if the user has 0.9.0.1 on the class path. It is not exactly tested on jenkins with it. @JingsongLi ran the tests with 'kafka.clients.version' set to 0.9.0.1, while it is not the samething as having different versions at build time and runtime, it helps. 

We thought post 2.0.0. release is a good time to upgrade the version (still works with 0.9.0.1).